src/createServer.js

```diff
@@ -1,11 +1,54 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { UserController } = require('./controllers/users.js');
+const { ExpensesController } = require('./controllers/expenses.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use(express.json());
+
+  const userController = new UserController();
+  const expensesController = new ExpensesController();
+
+  app.post('/users', userController.add.bind(userController));
```

Comment:
```suggestion
  app.post('/users', userController.add);
```
Better bind this inside constructor (I've added example)

But in ES6 class syntax you can define methods as arrow functions so this will always points to class itself.